### PR TITLE
Simple Web-App Running on Tomcat

### PR DESCRIPTION
**Pull request by ayushmalikofficial**
Simple Maven Web-App with Login-Logout Page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# authmodule
+# authmodule, Ayush Malik
```
Comment (vaibhavjain3) [inline README.md]: Let's remove the name.

File: authmodule/pom.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```
Comment (vaibhavjain3) [inline authmodule/pom.xml]: Let's move this dependency version as properties.

File: authmodule/src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE web-app PUBLIC
+ "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+ "http://java.sun.com/dtd/web-app_2_3.dtd" >
+
+<web-app>
+  <display-name>Archetype Created Web Application</display-name>
```
Comment (vaibhavjain3) [inline authmodule/src/main/webapp/WEB-INF/web.xml]: what's archetype?

File: authmodule/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<html>
```
Comment (vaibhavjain3) [inline authmodule/src/main/webapp/index.jsp]: indentation of xml

**Review by vaibhavjain3 — CHANGES_REQUESTED** [review]
please resolve these.

**Comment by ayushmalikofficial** [comment]
Pushed the necessary changes

File: README.md (hunk 1 above)
Comment (vaibhavjain3) [reply README.md]: Let's resolve the review comments with proper description once done.

File: authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+//		.password("password")
+//		.roles("ADMIN");
+	}
+	@Bean
+	public PasswordEncoder getPasswordEncoder() {
+		return NoOpPasswordEncoder.getInstance();
+	}
+	@Override
+	protected void configure(HttpSecurity http) throws Exception{
+//		http.authorizeRequests()
```
Comment (vaibhavjain3) [inline authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java]: Let's refrain ourself from adding commented out code unless it's really required and have proper commenting.

File: authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.security.crypto.password.NoOpPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+public class SpringSecurityConfig extends WebSecurityConfigurerAdapter{
+
+	@Override
+	protected void configure(AuthenticationManagerBuilder auth) throws Exception{
+//		auth.inMemoryAuthentication()
```
Comment (vaibhavjain3) [inline authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java]: Let's refrain ourself from adding commented out code unless it's really required and have proper commenting.

File: authmodule/src/main/resources/application.properties [hunk 7]
```diff
@@ -0,0 +1,2 @@
+spring.security.oauth2.client.registration.google.client-id=
```
Comment (vaibhavjain3) [inline authmodule/src/main/resources/application.properties:3]: Let's add little bit of comments here on what is client-id and secret here. and how can a new person can get it.

**Review by vaibhavjain3 — CHANGES_REQUESTED** [review]

File: authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java (hunk 5 above)
Comment (ayushmalikofficial) [reply authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java]: Changed // with /**/ comments. Added details related to In-Memory Authentication

File: authmodule/src/main/resources/application.properties (hunk 7 above)
Comment (ayushmalikofficial) [reply authmodule/src/main/resources/application.properties:3]: Added some comments related to the same including the link for official documentation on how to generate and use google client-id and secret. Will add more details soon.

**Comment by ayushmalikofficial** [comment]
The following changes have been pushed to the project:

- All HTML pages now follow VSCode standards
- Some documentation related to Client-ID and Client-Secret have been added in application.properties
- Comments related to In-Memory Authentications have been updated.

**Comment by ayushmalikofficial** [comment]
Upcoming Changes
- The controller in HomePage.java will return HTML pages instead of Strings.
- Google OAuth Module to be changed in the form of Sign-In with Google box.
- Authentication with Hibernate will be added.

File: authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java (hunk 6 above)
Comment (ayushmalikofficial) [reply authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java]: Format of comments has been changed. Details related to In-Memory Authentication have been added.

File: authmodule/src/main/webapp/index.jsp (hunk 4 above)
Comment (ayushmalikofficial) [reply authmodule/src/main/webapp/index.jsp]: All HTML pages now follow VS-Code standards.

File: authmodule/src/main/webapp/WEB-INF/web.xml (hunk 3 above)
Comment (ayushmalikofficial) [reply authmodule/src/main/webapp/WEB-INF/web.xml]: Updated the name with 'authmodule'

File: authmodule/pom.xml (hunk 2 above)
Comment (ayushmalikofficial) [reply authmodule/pom.xml]: Junit.version property has been added.

File: README.md (hunk 1 above)
Comment (ayushmalikofficial) [reply README.md]: Name has been removed.

**Pull request merged**